cmd/snowstorm/main.go

```diff
@@ -485,46 +366,42 @@ func getFlakesForJob(name string, depth int, gcsBucket *storage.BucketHandle) er
 	}
 	close(buildJobs)
 	wg.Wait()
-	glog.Infof("name %q finished (took %s)", name, time.Since(now))
+	glog.Infof("name %q finished (elapsed %s)", name, time.Since(now))
 	return nil
 }
 
 func Usage() {
 	fmt.Fprintf(os.Stderr, `
-Usage: %s [-interval <sec>] [-workers <sec>] [-min-flake-failures <sec>]
+Usage: %s [-config <string>]
 `, os.Args[0])
 	flag.PrintDefaults()
 }
 
 func main() {
-	intervalFlag := flag.Uint("interval", 3600, "interval in seconds to run scrapper")
-	workersFlag := flag.Uint("workers", 20, "number of worker threads to use")
-
+	configPath := flag.String("config", "", "a path to a config file to use")
 	flag.Usage = Usage
 	flag.Parse()
-
-	if intervalFlag != nil {
-		interval = time.Second * time.Duration(*intervalFlag)
-	}
-	if workersFlag != nil {
-		numWorkers = int(*workersFlag)
+	var err error
+	config, err = types.ParseConfig(*configPath)
+	if err != nil {
+		glog.Fatalf("error reading config %q: %v", *configPath, err)
 	}
 
 	ctx := context.Background()
-	gcsClient, err := storage.NewClient(ctx)
+	gcsClient, err := storage.NewClient(ctx, option.WithoutAuthentication())
```

Comment:
Not sure we want this

Reply:
@stevekuznetsov this will guarantee you won't accidentally pass your local token